src/app/(admin)/settings/components/settings-sidebar.tsx

```
@@ -0,0 +1,113 @@
+        { id: 'organization', label: 'Organization', icon: BuildingOfficeIcon },
+        { id: 'notifications', label: 'Notifications', icon: BellIcon },
+        { id: 'security', label: 'Security', icon: ShieldCheckIcon },
+        { id: 'integrations', label: 'Integrations', icon: PuzzlePieceIcon },
+        { id: 'analytics', label: 'Analytics', icon: ChartBarIcon },
+        { id: 'billing', label: 'Billing', icon: CreditCardIcon },
+        { id: 'behavior', label: 'Behavior', icon: CommandLineIcon },
+        { id: 'localization', label: 'Localization', icon: GlobeAltIcon },
+    ];
+
+    const activeSectionData = sections.find(section => section.id === activeSection);
+    const ActiveIcon = activeSectionData?.icon || Cog6ToothIcon;
```

ESLint

[error] 'ActiveIcon' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)